pages/models.py

```diff
@@ -615,6 +620,29 @@ class Adopters(Page):
     parent_page_types = ['pages.HomePage']
 
 
+class Organizations(models.Model):
```

Comment:
I understand this might be SF terminology, but this is also how the frontend will be querying for Adopters, and it's confusing.